### PR TITLE
[vcpkg-tool-meson] update meson and see what breaks

### PR DESCRIPTION
**Pull request by Neumann-A**

### CODE REVIEW

**Comment by JackBoosY** [comment]
Any progress in this PR?

**Comment by Neumann-A** [comment]
I don't consider this a priority. If you have time to fix the regression please do so.

**Review by JackBoosY — APPROVED** [review]

File: ports/vcpkg-tool-meson/vcpkg-port-config.cmake [hunk 1]
```diff
@@ -0,0 +1 @@
+set(MESON_SCRIPT "${CMAKE_CURRENT_LIST_DIR}/../../tools/meson/meson.py")
```
Comment (ras0219-msft) [inline ports/vcpkg-tool-meson/vcpkg-port-config.cmake:1]:
```suggestion
set(SCRIPT_MESON "${CMAKE_CURRENT_LIST_DIR}/../../tools/meson/meson.py")
```
Assuming you were trying to interface with `vcpkg_find_acquire_program`?

**Review by ras0219-msft — APPROVED** [review]
LGTM once you've had a chance to look at my comment. Give me a thumbs up if you're happy with the PR.

File: ports/vcpkg-tool-meson/vcpkg-port-config.cmake (hunk 1 above)
Comment (Neumann-A) [reply ports/vcpkg-tool-meson/vcpkg-port-config.cmake]: yeah. I originally wanted to set `MESON` but that requires `vcpkg_find_acquire_program(PYTHON3)` which seems to not work in `port-config`s due to include order in `ports.cmake`

**Comment by ras0219-msft** [comment]
Thanks!

**Pull request merged**